W-21862451: omni first draft dm (#402)
mulesoft/docs-general: modules/ROOT/pages/exp-services-add-semantic.adoc

To add semantic services, follow these steps:

. Define the semantic service your organization uses for contextual matching.
. Associate that semantic capability with the relevant enhanced experience (for example, LLM-related routing paths where supported).
. Operate the service as part of your broader governance and monitoring model so behavior remains reliable as traffic and use cases evolve.

== Before You Begin

Before getting started, make sure you have:

* An Anypoint Platform account.
* Any of these permissions:
+
--
** Exchange: Exchange Contributor
** Exchange: Exchange Administrator
** Exchange: Exchange Creator
--
+
For more information, see xref:exp-home-start.adoc#permissions[Enhanced Experience Permissions].

== Create a Semantic Service

. In the navigation, under *Portfolio*, expand *LLM Proxies*, and select *Semantic Services*.
. Select *Add Semantic Service*.
. Select the scale model that matches your traffic profile and operational needs.
+
* *Basic scale*
+

> Choose *Basic scale* for faster managed setup. This scale is best for smaller topic sets (up to 6 topics and 10 utterances per topic). Uses managed internal LLM proxy configuration.

Reviewer: ```suggestion
Choose *Basic scale* for faster managed setup. This scale is best for smaller topic sets (up to 6 topics and 10 utterances per topic) and uses managed internal LLM proxy configuration.
```

+
* *Advanced scale*
+

> Choose *Advanced scale* when you need greater scale and external infrastructure control. This scale is best for larger utterance sets per topic. Uses external semantic infrastructure, including an embedding API connection and a vector database connection.

Reviewer: ```suggestion
Choose *Advanced scale* when you need greater scale and external infrastructure control. This scale is best for larger utterance sets per topic. It uses external semantic infrastructure, including an embedding API connection and a vector database connection.
```

> Semantic service setup defines the embedding connection used for contextual matching. Provide a value for these fields:

Reviewer: ```suggestion
Semantic service setup defines the embedding connection used for contextual matching. Provide values for these fields:
```
